devicon.json

```diff
@@ -4383,7 +4383,32 @@
         "color": "#F18803",
         "aliases": []
     },
-	{
+    {
+        "name": "ohmyzsh",
+        "altnames": [
+            "omz",
+            "omzsh"
```

Comment:
never spaces in names and `ohmyz.sh` is the url of there website nothing to do with the product / tool

Reply:
1. The `altnames` array is used for search on the website. It doesn't really matter if there are spaces since it's formatted with UTF8 which supports spaces and special characters. In fact it's good that there are spaces, since `Oh My Zsh` is a common way to search for it, which is exactly what the altnames is used for.

2. `ohmyz.sh` is debatable, but imo it's appropriate. My reasoning is that it's a common way to search for Oh My Zsh. I might be wrong here though, so feel free to come with more input on this